tests/TestCase/View/CellTest.php

```diff
@@ -110,7 +110,6 @@ public function testCellImplictRenderWithError(): void
 
         $cell = $this->View->cell('Articles::teaserList');
         $cell->viewBuilder()->setTemplate('nope');
-        (string)$cell;
```

Comment:
The string casting is to implicitly trigger the `Cell::render()` call, it's needed.